src/models/models.py

```diff
@@ -86,11 +86,12 @@ class SpeakerListEntry(Base):
     __tablename__ = "speakerlistentries"
 
     # id
-    speakerlistentry_id = Column(String, primary_key=True, index=True)
+    speakerlistentry_id = Column(Integer, primary_key=True, index=True)
 
     # foreign ids
-    speakerlist_id = Column(String, ForeignKey("speakerlists.speakerlist_id"))
-    participant_id = Column(String, ForeignKey("participants.participant_id"))
+    speakerlist_id = Column(Integer, ForeignKey("speakerlists.speakerlist_id"))
+    participant_id = Column(Integer, ForeignKey("participants.participant_id"))
+    spoke = Column(Boolean, default=False)
```

Comment:
It's probably best for us to keep track of everyone who has spoken. NMUN had the names get crossed off as they went. May even help us keep some data.